nemo/collections/tts/data/vocoder_dataset.py

```diff
@@ -71,7 +132,7 @@ class VocoderDataset(Dataset):
     def __init__(
         self,
         dataset_meta: Dict,
-        sample_rate: int,
+        sample_rate: int = 24000,
```

Comment:
Nitpick: other classes in the TTS collection either provide no default sample_rate or defaults to 22050.

Reply:
Default for our audio codec models is 24000. Did we have a 22050 Hz model?

Reply:
No, but this is a data loader for all TTS vocoders. So I suppose I would favor not having a default sample rate.

Reply:
updated